core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -143,6 +144,7 @@ private ReadWriteLock getPageLock(PageId pageId) {
 
   @Override
   public boolean put(PageId pageId, byte[] page) throws IOException {
+    LOG.debug("put(pageId={},page={} bytes)", pageId, page.length);
```

Comment:
will the `PageId` instance already print out `PageId`? Will `pageId=` be redundant in the log?